man/build/build-manpages.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env -S python3 -I
```

Comment:
It's probably best to move this script to the top-level `./scripts/` directory and add it to that `Makefile.am`.  This directory already contains a variety of similar support scripts.